woocommerce/woocommerce
# [Accessibility] Prevent the variation list from being displayed as an HTML list if there is only one attribute on the Mini Cart Modal

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes #51903 .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Activate a block-based theme.
1. Make sure your store has these two types of products:
    1. One product with one variation (e.g Color)
    2. Another product with two or more variations (e.g Color and Size)
2. Add both products to the Cart.
3. Open the Mini Cart Slide Modal.
![Screenshot 2024-11-18 at 19 15 40](https://github.com/user-attachments/assets/91db4396-a3bc-48bb-b3b0-2cf0c7adee9e)
4. Verify that when the product has only one variation, the attributes on the Mini Cart is not an HTML list.
![Screenshot 2024-11-18 at 19 18 18](https://github.com/user-attachments/assets/9d69721c-ac28-4f1d-9b45-a69b4e972dc8)
5. Verify that if the product has more than one variation, the list of attributes is an HTML list (`<ul>` + `<li>`)
![Screenshot 2024-11-18 at 19 21 01](https://github.com/user-attachments/assets/63129531-a834-4847-b107-f66948df8a93)




### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
